src/components/comments.js

```diff
@@ -34,6 +37,29 @@ const userRef = {
     },[]);
 
 
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        setIsLoading(true);
+        setErr(null)
+        fetch(`https://hannybees-news-app.herokuapp.com/api/articles${articleIdRoute}/comments`, {
+    method: 'POST',
+    body: JSON.stringify({
+    username:'cooljmessy',
+    body: `${newComment}`,
+    }),
+    headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+    },
+    }).then((response) => response.json())
+    .then((json) => setCommentData((currData)=>[json,...currData])).catch((err)=>{
```

Comment:
I dont know your intention but this isn't rendering optimistically, your handling it nicely with your isLoading. I think its a  perfectly reasonable use of non-optimistic rendering 👍